terraform/terraform-0.13.patch

```diff
@@ -0,0 +1,32 @@
+From: Kurt Garloff <scs@garloff.de>
+Subject: Adjust terraform to work with version 0.13
+
+We don't use any new features from 0.14 yet, so we can easily change
+the syntax to work with 0.13. The way providers are specified has changed,
+thus this is only a patch and not integrated into the main branch.
```

Comment:
Why? Makes no sense to me to enable per patch 0.13. We should always use the latest version of Terraform. This is also tested in the CI.